recipes/pingouin/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pingouin" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1f585181b7f6722d61b8b1155a0bddf1bdb6de7caecd2a78ba5ace7e32ecd13a
+
+build:
+  noarch: python
+  number: 0
+  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
+  # Once resolved, it should be removed.
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
```

Comment:
I think you need the python version selector here too IIUIC. Looking at the circleCI build, it used python 2.7 to install the recipe. I think we want to force it to use python 3.6 or greater to install.

Reply:
I was following the [building `noarch` packages](https://conda-forge.org/docs/meta.html#building-noarch-packages) section of the docs:

> Note that while `noarch: python` does not work with selectors, it does work with version constraints, so `skip: True  # [py2k]` can sometimes be replaced with a constrained python version in the build/run subsections: say `python >=3` instead of just `python`.

It only mentions `build` and `run` there, not `host`. So I wonder whether the docs need updating too?

Reply:
I just checked the CircleCI log file, and the package was _not_ installed into a Python 2 environment. It was correctly built as a `noarch` package and then installed into a Python 3.7 environment.

Reply:
Version constraint added as requested, recipe should be good to merge now.

Reply:
Yep. I think the docs need updating. The host section is new in conda build.